src/applications/personalization/rated-disabilities/actions/index.js

```diff
@@ -22,21 +20,29 @@ export function fetchRatedDisabilities() {
       const errorCode = response.errors[0].code;
       if (isServerError(errorCode)) {
         recordEvent({
-          event: `${DISABILITY_PREFIX}-list-load-failed`,
-          'error-key': `${errorCode} internal error`,
+          event: `api_call`,
+          'error-key': `${errorCode} server error`,
```

Comment:
in the payments.js recordEvent it looks like the error key is `${errorCode}_server_error` should the error key for this and the client error also use underscores for consistency? I'm unfamiliar with the GA stuff in general, so just wondering if that would make errors easier to search in the GA dashboard or whatnot.